components/output/mcp4461.rst

```diff
@@ -0,0 +1,188 @@
+MCP4461 Component
+=================
+
+.. seo::
+    :description: Instructions for setting up MCP4461 outputs on the ESP.
+    :image: mcp4461.jpg
+
+.. _mcp4461-component:
+
+The MCP4461 output component enables the use of an 8‑bit external quad digital potentiometer/rheostat via I²C. See the `MCP4461 Datasheet <https://www.microchip.com/en-us/product/mcp4461>`__ for more information.
+Device default address is ``0x2C``.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+
+    # Set a global I²C connection
+    i2c:
+      sda: 21
+      scl: 22
+      scan: true
+
+    # Set the component with default (address: 0x2C / global I²C)
```

Comment:
Examples must be minimal and only include yaml relevant to the component/platform itself.
```suggestion
```